src/components/typography-control/index.js

```diff
@@ -14,6 +14,9 @@ import { i18n } from 'stackable'
 import { SelectControl } from '@wordpress/components'
 
 const TypographyControl = props => {
+	const {
+		placeholder = '21',
+	} = props
```

Comment:
Please move this in the defaultProps on the bottom of the file along with the other values.